### ISSUE TITLE
Transition from Travis CI to Github Actions

### ISSUE THREAD

**Issue by smjenness** [EpiModel/EpiModel-Gallery#31]
Github actions needs to install all the relevant packages and then run this script:

https://github.com/EpiModel/EpiModel-Gallery/blob/master/test.sh

**Comment by smjenness** [comment]
@AdrienLeGuillou Do you think this would be relatively straightforward to implement? If we could borrow the R CMD Check workflow from our statnet packages?

**Comment by martinamorris** [comment]
@krivit transitioned all the statnet packages -- so you might want to 
check in with him.

**Comment by smjenness** [comment]
@martinamorris This is not a package, thus the (hopefully slight) wrinkle.

**Comment by AdrienLeGuillou** [comment]
It is now implemented.

I had to remove the ".Rprofile" file that initialized `renv` as `renv` is not compatible with changing the working directory (strategy used in "test.sh").

I also corrected some errors in the examples and updated the DESCRIPTION file to import `EpiModel >= 2.0` and `dplyr`.

The model "2018-08-ObservedNetworkData" is not working for now when using `EpiModel > 2.0.5` I will open another issue for it.
